tests/unit/test_bson_fields.py

```diff
@@ -20,12 +20,19 @@ def test_datetime_non_naive():
     class ModelWithDate(Model):
         field: datetime
 
-    with pytest.raises(ValueError):
+    try:
         ModelWithDate(field=datetime.now(tz=pytz.utc))
+        ModelWithDate(field="2018-11-02T23:59:01.824Z")
+        ModelWithDate(field="2018-11-02T23:59:01.824+00:00")
+    except ValueError:
+        pytest.fail("UTC should be acceptable, but is not")
```

Comment:
I think, you can move those to a new test and for example check that the field value is correct for all those

Reply:
Will do!